cpp/include/cuvs/neighbors/ivf_pq.hpp

```diff
@@ -2896,7 +2894,7 @@ void set_centers(raft::resources const& res,
  */
 void extract_centers(raft::resources const& res,
                      const index<int64_t>& index,
-                     raft::device_matrix_view<float, uint32_t, raft::row_major> cluster_centers);
+                     raft::device_matrix_view<float, int64_t, raft::row_major> cluster_centers);
```

Comment:
We might need to change the signatures of ivf-pq getters to int64_t instead of uint32_t, right? For example things like `list_sizes`.

Reply:
yes so we have `n_lists()` for example https://github.com/rapidsai/cuvs/blob/ba67db12dcd2b4f894e9b0693c11e85dff4c3626/cpp/include/cuvs/neighbors/ivf_pq.hpp#L406

Do we want to change this signature to `int64_t`?

Reply:
In other words, what I am trying to say is that we might be considering an index-wide migration to `int64_t` from `uint32_t` for all extent types in IVF-PQ.

Reply:
@tfeher had made of aware of plans for this, but maybe that is outside the scope of this PR.